regexes/device/mobiles.yml

```diff
@@ -1540,6 +1539,9 @@ Sony:
       model: 'Xperia X'
     - regex: '(?:Sony(?:Ericsson)?)?F5321'
       model: 'Xperia X Compact'
+    - regex: 'E5506'
```

Comment:
Could you please change that to the previous regex: `'(?:Sony(?:Ericsson)?)?E5506'`